### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by wrznr**

### CODE REVIEW

**Comment by tboenig** [comment]
Hallo wrznr,

it's great, I can tocrify now install without problems on my Windows notebook (Bash on Ubuntu on Windows). The description is short and easy to understand. But how can I use the tool now? Can you explain that?

**Comment by wrznr** [comment]
@tboenig Many thanks for your suggestion. Please check if your a satisfied with the section on `tocrify`'s invocation.

**Comment by wrznr** [comment]
Fixes #12

**Comment by tboenig** [comment]
Hi wrznr,
the Readme.md is fantastic. The installation and a first test on my Linux subsystem on Windows worked.
Thanks for all.

File: README.md [hunk 1]
```diff
@@ -41,5 +48,26 @@ Finally, `tocrify` itself can be installed via `pip`:
...
+## Invocation
+`tocrify` comes with a help message explaining its usage:
+```console
+(env) $ tocrify --help
+Usage: tocrify [OPTIONS] METS
+
+  METS: Input METS XML
+
+Options:
+  -o, --out-dir PATH         Directory for storing the updated OCR files
```
Comment (tboenig) [inline README.md]: I would like to see a hint that the directory has to exist!

**Review by tboenig — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (kba) [reply README.md]: @wrznr Could also do

```python
if not os.path.exists(out_dir):
  makedirs(out_dir)
```

before writing out_filename

**Review by tboenig — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (wrznr) [reply README.md]: @kba Sorry, I don't like/want my software to create directories.

**Pull request merged**